CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ project(az LANGUAGES C)
 enable_testing ()
 
 include(eng/cmake/global_compile_options.txt)
-include(cmake-modules/create_map_file.cmake)
+include(create_map_file)
```

Comment:
How does this work to find the right path to the module?

Reply:
on line 6 we do:
```cmake
list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake-modules")
```
That line makes all modules to become available to to `include(module)`

Reply:
I see. And what about the `.cmake` file extension? Does cmake assume that will be the file extension?

Reply:
yes, my guess is that any file with .cmake extension in cmake module path can be imported directly with no extension (that's how all other modules are imported). see: https://cmake.org/cmake/help/v3.12/variable/CMAKE_MODULE_PATH.html